webapp/accounts/urls.py

```diff
@@ -0,0 +1,23 @@
+from django.conf.urls import url
+from .import views
+from django.contrib.auth.views import (
```

Comment:
Why are we importing views from the `auth` app?
Why is this needed here? It seems like we could have just had a login, logout and profile page to suffice.